docs/src/_layouts/docs.njk

```diff
@@ -0,0 +1,36 @@
+---
+layout: base
+styles:
+  - '/@root/styles/docs.scss'
```

Comment:
Manually load CSS for docs only when the docs layout is applied. Frontmatter is applied by the base layout